third_party/Wikipedia/wikipedia-search-cookbook.ipynb

```diff
@@ -369,12 +435,28 @@
     "            max_searches_to_try=max_searches_to_try,\n",
     "            temperature=temperature,\n",
     "        )\n",
-    "        information = extract_between_tags(\"information\", retrieval_response)[-1]\n",
+    "\n",
+    "        # Try to extract information tags, handle case where none exist\n",
```

Comment:
When `<information>` tags are missing you use the entire retrieval response as information. This could include all the scratchpad content and search quality reflections. Any reason why this is necessary? If no <information> tags are found I would think there's an error in the response